docs/components/cloud-console/manage-organization/manage-users.md

```diff
@@ -12,24 +12,22 @@ If several users need access to the same Zeebe cluster, all users can be assigne
 
 ## Users
 
-Under this setting, members of the current organization can be managed. A user can have one of the following roles:
+The first user in an organization is usually the owner of the organization. An owner has all rights in an organization and can manage all settings accordingly. An organization cannot have more than one owner.
 
-- **Owner**: Owner of the organization (currently limited to one user and cannot be changed by the user.)
-- **Admin**: Restricted rights for user management.
-- **Member**: Can manage Zeebe Clusters, Client, and use Operate.
+If the owner of the organization has to be changed for a certain reason, this can be done in the user administration. The current owner selects another member of the organization and chooses **Assign as Owner** from the menu. In the dialog that appears, you can select which new roles are to be assigned to the current owner.
 
-The following table illustrates the rights of each role:
+### Roles and Permissions
 
-|                              | Owner | Admin | Member |
-| ---------------------------- | ----- | ----- | ------ |
-| Manage Zeebe clusters        | X     | X     | X      |
-| Manage clients               | X     | X     | X      |
-| Use Operate                  | X     | X     | X      |
-| Users: Manage members        | X     | X     |        |
-| Billing: Manage reservations | X     | X     |        |
-| Billing: Request paid plan   | X     | X     |        |
-| Users: Manage admins         | X     |       |        |
+In addition to the Owner, the **Admin** role is available as a second role with comprehensive rights. The Admin role has the same rights as the Owner, with the difference that an Admin cannot manage other Admins.
+
+The following roles are additionally available, providing dedicated rights for specific elements in Camunda Cloud.
+
+- **Operations Engineer**: Full access to Console and Operate, except Cluster deletion privileges
+- **Task User**: Full access to Tasklist and read-only access to Clusters
+- **Developer**: Full access to Console, except deletion privileges. Full access to Operate, and Tasklist
+- **Visitor**: Read-only access
+- **Analyst**: Full access to Optimize and read-only access to Clusters
 
 Users are invited to a Camunda Cloud organization via their email address, which must be accepted by the user. As long as the invitation has not been accepted, the user remains in the `Pending` state.
 
-People who do not yet have a Camunda Cloud account can also be invited to an organization. To access the organization, however, the invited individual must first create a Camunda Cloud account.
\ No newline at end of file
+People who do not yet have a Camunda Cloud account can also be invited to an organization. To access the organization, however, the invited individual must first create a Camunda Cloud account.
```

Comment:
...by following the instructions in the invitation email?

Reply:
fixed